draft-packages/form/KaizenDraft/Form/Primitives/Input/styles.scss

```diff
@@ -125,6 +125,11 @@ $input-with-icon-padding: $input-icon-size + $ca-grid * 0.75;
   .endIconAdornment {
     @include vertically-center-icon;
     @include ca-position($end: $ca-grid / 2);
+
```

Comment:
You need to add some rtl support to handle to handle the icon and text placement to ensure they don't overlap

![Screen Shot 2022-04-22 at 12 19 13 pm](https://user-images.githubusercontent.com/36558508/164584107-5215dc75-703b-4004-97e3-72e626551f4c.png)

Reply:
We aren't handling RTL in this iteration :)

Reply:
gotcha 👍